content/blog/2016-11-16-interview-with-canonical.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Interview with Canonical
+created_at: 2016-11-16
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, Canonical talks
+about how they are transitioning to Prometheus.*
+
+## Can you tell us about yourself and what Canonical does?
+
+[Canonical](http://www.canonical.com/) is probably best known as the company
+that sponsors Ubuntu Linux.  We also produce or contribute to a number of other
+open-source projects including MAAS, Juju, and OpenStack, and provide
+commercial support for these products.  Ubuntu powers the majority of OpenStack
+deployments, with 55% of production clouds and [58% of large cloud
+deployments](
+https://www.openstack.org/assets/survey/April-2016-User-Survey-Report.pdf).
```

Comment:
Maybe we can link directly to page 47 of the article?

```
https://www.openstack.org/assets/survey/April-2016-User-Survey-Report.pdf#page=47
```

But feel free to ignore this as it's not that important for the article and I'm unsure whether this works consistently across browsers.